Makefile

```diff
@@ -847,7 +847,7 @@ $(RCPTS)/fvtr.rcpt: $(RCPTS)/package.rcpt
 	   AT_WD=$(AT_WD) \
 	   AT_PREVIOUS_VERSION=$(AT_PREVIOUS_VERSION) \
 	   AT_DIR_NAME=$(AT_DIR_NAME) \
-	   ./fvtr.sh -f \
+	   ./fvtr.sh -f -d \
```

Comment:
I made it this way because the requirement is that the builtbot should print detailed logs specifically for tests that failed. The alternative to placing -d as a default in the makefile would be creating a new option for make test that makes it pass the -d option to fvtr.sh, and then the buildbot passes the appropriate option to make test by default. Would that be preferable? Other suggestions?